File: js/euklidesGame.js
function euklidesGame(A, B) {
    var count = 0

    if (A == B) {
        count = A + B
        return count
    }

    if (A > B) {
        difference = function (A, B) {
            return A - B
        }
        count = A - difference(A, B)
        return count
    }

    if (B > A) {
        difference = function (A, B) {
            return B - A
        }
        count = B - difference(A, B)
        return count
    }
}
console.log(euklidesGame(2, 4))

File: c#/euklidesGame.cs
using System;

namespace ConsoleApp19
{
    class Program
    {
        static int euklidesGame(int A, int B)
        {
            int count = 0; 

            if (A == B)
            {
                count = A + B;
            }

            if (A > B)
            {
                int difference = A - B;
                count = A - difference;
            }
            if (B > A)
            {
                int difference = B - A;
                count = B - difference;
            }
            return count;
        }
        static void Main(string[] args)
        {
            Console.WriteLine(euklidesGame(2, 4));
        }
    }
}
